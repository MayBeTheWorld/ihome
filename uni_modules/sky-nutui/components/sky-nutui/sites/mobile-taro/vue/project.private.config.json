{
  "setting": {
    "compileHotReLoad": true
  },
  "description": "项目私有配置文件。此文件中的内容将覆盖 project.config.json 中的相同字段。项目的改动优先同步到此文件中。详见文档：https://developers.weixin.qq.com/miniprogram/dev/devtools/projectconfig.html",
  "condition": {
    "miniprogram": {
      "list": [
        {
          "name": "feedback/pages/notify/index",
          "pathName": "feedback/pages/notify/index",
          "query": "",
          "launchMode": "default",
          "scene": null
        },
        {
          "name": "pages/index/index",
          "pathName": "pages/index/index",
          "query": "",
          "scene": null
        },
        {
          "name": "exhibition/pages/countdown/index",
          "pathName": "exhibition/pages/countdown/index",
          "query": "",
          "scene": null
        },
        {
          "name": "feedback/pages/collapse/index",
          "pathName": "feedback/pages/collapse/index",
          "query": "",
          "scene": null
        },
        {
          "name": "business/pages/signature/index",
          "pathName": "business/pages/signature/index",
          "query": "",
          "launchMode": "default",
          "scene": null
        },
        {
          "name": "exhibition/pages/collapse/index",
          "pathName": "exhibition/pages/collapse/index",
          "query": "",
          "launchMode": "default",
          "scene": null
        },
        {
          "name": "dentry/pages/textarea/index",
          "pathName": "dentry/pages/textarea/index",
          "query": "",
          "launchMode": "default",
          "scene": null
        }
      ]
    }
  },
  "projectname": "NutUI-Taro"
}