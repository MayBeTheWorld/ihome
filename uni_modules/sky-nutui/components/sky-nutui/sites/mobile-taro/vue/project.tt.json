{
  "miniprogramRoot": "./",
  "projectname": "@nutui/nutui-taro-mobile",
  "appid": "touristappid",
  "setting": {
    "es6": false,
    "minified": false
  }
}
