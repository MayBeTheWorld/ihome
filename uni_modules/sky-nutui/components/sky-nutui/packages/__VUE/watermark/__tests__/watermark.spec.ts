import { config, DOMWrapper, mount } from '@vue/test-utils';
import Watermark from '../index.vue';
import Cell from './../../cell/index.vue';
import { nextTick, toRefs, reactive } from 'vue';

test('render fullPage props ', async () => {
  const wrapper = mount({
    components: {
      'nut-cell': Cell,
      'nut-watermark': Watermark
    },
    template: `
      <nut-cell>
        <nut-watermark  font-color="#fa2c19" content="nut-ui"></nut-watermark>
      </nut-cell>
    `,
    setup() {}
  });
  await nextTick();

  expect(wrapper.findAll('.nut-watermark').length).toBe(1);
  expect(wrapper.findAll('.nut-watermark-full-page').length).toBe(1);
});
test('should render watermark', () => {
  const wrapper = mount({
    components: {
      'nut-cell': Cell,
      'nut-watermark': Watermark
    },
    template: `
      <nut-cell>
        <nut-watermark :fullPage="false" font-color="#fa2c19" content="nut-ui"></nut-watermark>
      </nut-cell>
    `,
    setup() {}
  });
  expect(wrapper.html()).toMatchSnapshot();
});
