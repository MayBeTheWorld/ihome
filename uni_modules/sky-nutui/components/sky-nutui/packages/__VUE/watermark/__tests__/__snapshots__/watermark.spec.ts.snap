// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render watermark 1`] = `
"<view class=\\"nut-cell\\" style=\\"border-radius: px;\\">
  <view class=\\"nut-watermark\\" style=\\"z-index: 2000; background-size: 144px; background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJAAAABwCAYAAADxAvk6AAAABmJLR0QA/wD/AP+gvaeTAAAP0ElEQVR4nO2df3QU13XHv/fNzqzELxsQaGelYBVjg9hZQYzblKZx4xCHOIndpA34FwEHDJw07Uli55wkbdqctglue5ImOU5iBP5RB9uYEh+cJk3iE2O7dhrbGByDdgHbsk1taVcYDDEGtDuzO7d/aIVWM7O7s9oRWsH7/OPj++bduRzdnbn3vXfvECTnNKYR2QrQ9Q7xa2oifQkBduX5+mYAtxTLGJTWwqmLaA8sEaSxkvqDgE4P8eycEV3iSwGLH7p1sm5m9U8AgHSgcxw10fckgP3uEV7vZ76W7H0RwB6PofWAdKC6hRdBDU4Z7nKL6Fpub9L9zaeNThEBV2XizbOlA9UZZlxfZMb0TiurHwzKiVQ1fC+A08OlrOYUdbWv+bAeAnDCIRZki7XSgeoABoRp6F8wDX0/GLtBWAdgtmlGPx6Efnrx0O8BPOxx37Xs4y1EySMnAbrfJSeslg5UBxSyoc8AaB82YPO6wO7B5BVMX5SPRz7qZz5DeM2fKR2oXiBscokIH80YMy8OQr2aTP0vQF1OOReC4UqEEz37AHrWKZcOVCeodu5BuOMMIii+4hRfeDgpM328v12/yN982/UUkg5UJwzEGXjQJQfWBBZMa/1bAJxyiBWh0Gd9zZ8U2gbgWLFMOlAdwVDu9BA3m2bkk0Hopz3H3wF4m8fQOj9OSs/09AMYFkxLB6ojwomefSB6zikn21+c4gt2r0wTWDfN6Mf8TcfGgf8MIB2o/nBnO0QfyixouTQI5VqybxeAF1wDtr+V6XAifQCM3wz+v3SgOkOdJB4CcNwhJsrzLV7XjwjvjG9pf0fzH/iZzmLIyaUD1RkDcQa7F+3Aq7mtrSGIe6h27gF4rCwLW/hyUs2eth3AEWAcOhDPmRMeaxtGG7Y9d9Cnm5PNvwhCfyHj2+oxtNpXMJ1Mmgy6DxhnDpSNR+dZDaee4mVQxtqW0SS8vy8JDMUZgxD7i1P8wES/cukHImYm8ue+5iO3EQCPGwfKGno7mJ8A8Ef5A/6W38cz7H2O54rs/EgsGP35fV5yIvLlpA2Jt14F0RPjwoHMWMsCAp4iIAL4X34fz2iZiWfijGJIoUCC6bCm9JUYWuI342O2O+vegU53tLSC7J0AmgZlDPpYfyw6awzNGnWouzsLxhbXAGMVL25trPkGs1PZUremPK/1o0IL9+2oewdqPN14BO6zu4pwnNM9F7FDohNFi3YFploncstrVt4dmVZqiMA3+0lWxsWZaOruzjLxvU65EPYa/iBCY2FTkJTLehr29r4M0JOuAar9FZ6zxCVlhpvMxtN/6UdP3TsQALAd2gzHL5GZouZR/Zqy8zqaZ7Ixs3lUjRsB3NbWkI3rN5kx/SnLjG4uey1s13FSgBabsZYFNRlB9ofLDvvM+MaFAzUke7oBPOYx5PpHMkBWLHKlaUQesmzxpsXKV0bfQn9kFrRcahrRb1uTsj3EuB+ED4B5OcdnTS01R+PpjwA47Bog+8sjtYMBYlDRa9B9zgfAFVlDb/eQD2NcOBAAsOcm4MDBbgDgeS3Trbh+q2XoB5jocYCuA6CBAgo6RwjHYlrWiCw349GdIm8fBPg2ANOLLmm02FpZaj4lkyYzXK9wANdnYq1zRmKTGYteD8bQcoCNvwLwtuvePrLdceNAWkP6vxhwpp5CQHzTNPQtVsjuYcZ3AMx1XDPNejf/6bNk5hky8ebZ2Zh+u0XH3iDQNjB/CAB5Xkwom/XkQ8oPADizppCg/J1cSmcJuL1JF8L+dtHNu7T9qd+BcZ/H5Ssr/fjGjQPRHlgA7nENMG4AsAJAuX2iy0fJrOGmfBChrKF/KmvovxIsXiHCVwGUi8EOM3B7nuyysdyEvT29IPzYY+jDOSNym2/73huZYSnqT5kpOiQciLFshTZhKM7MM+jnAvZKtPaY5XRW5b1jTTYW+TQRbfd5+XEQ7mPGpnAifWA07eqPRWcJ4BYh7DXD/jjeMJgfZ4FOzZ7+U0omy/6BBsnEWucIyicBaG59uFVNpr9P7pT/DLl45Grbph+B0FY085CanTiPuruzAGDG9QcZeNW2aXNjMvWGH7vq3oF4btNkS1NvBLAejPf6mPEMGJ3qlNB/Fk7QjY5dy6DkDzRfnYeynsBXAxX3544y6D9Yoc0D6Xn1WPHIPzDTP3obxDtZiDu0/swT1H3sBC9ubbRO8KUEXsrEy+B+CjOBlqqJ1K9HYssgdetAZkf0skJZy40AJleewdsYoQ0D1QOjx6m574lqmrWGmdYCeE9ls/A0C9qo9U94ePCXPlJ4EVQrq+8CsLDCpRmUf6WDGH+vJtPfrMUeAPW5EMcdzTMtm59DVfaJntFyHgZELqZfBcJ6Ru4aZqpk13EAW5i5M5zs86hLHxm0B1b/Qv6UkqP/AVBuK6ec8zARf0NN9NXsPEAdP4EsQ3+YAa/zLycBvAzgMof8qJqZ2Frrr7wYNmY25yj0WWZeC2B25Rn0LMjuVCeFto3m6zPT3nqJUPI74ecJWAQBvXmmzzUkUz8LypYxcyBua2uwJmaWg2htDnTjhETqzeJxy4h+hMGPFoleBNCpWtYD/eHwBSHbfh2OJxQTVoS70g/UZBdAuVjzlUzKeoA/CXfQ6uQEQA+AqVNL9u6t5d7VwAvbLrRyme8AVLFujIAeMN0Rypl30ktH3w3SjrPuQNl4dB7ZvA6EVQCmAQAz/VM4mfpG8XUMCMvQ94JpN5R8p7bv8LDVUiumP8KE4YefGE9ryfQVI7GLL4025TR7FTOtA8HPcYY9YHSqyG0tnPAbE7KxyHwIuoEYSwmIMjAVQB+YX2fQ8wTeoSb7ni+XodXCWXcg09DvArDGYURvqCndRk8iVyxnQJTqopWLR662mX7hlLPNRuFEX5V2RVcDfHeFy05h4Chop5ZI7672HuciZ38hkdldXgu0mG+7O1GUa8GmdPU9CsYh1xxBvs6yOFEni61wV0MMcpCIvqqyMktLpNdK5xnirDtQrXVJgxBgE7kbJwFYxYuiE6q1i57p6QdzcfyUJdB2IrpKS6Tb1a7Uv1Ky51hJBecpY7OVQR4bo1XUJQ0S4tzdAFkO8YWWaY/owBUzNmKgHdwXVVJ1NZFarnalvE4BSAqMSRbGsRmTLAr1ApgyTA5sCCfSf1eNLsvQf8KA4/ATPaslUotrNlRSkTF5AtVal+TQ5lG9wH9szo/62PaQ1IpvB2JAWLHo+wO7Mwuvxo2RajtRhBKpxwDudg2I4Lp7SUpT0YHYmNlsxaNfsQz9FSb+TVB1SYX2sbuccmKq6g9PABMLr2OhN/HcJh97aJJaKOtAZkzvtBB6k5n/BYWlfBLV/YHLQu72sQCWZNpbyx34dhHS7HvhPnA12dLUG0Zsm8QXlZ5AJwF2xiQrR5Ime1GyE4ViV7WWQ7/rOwLwDtcA43M1mCfxQVkHsgc6ZjmXwC+0MvayIG4eZCeKEi39F5qGflZOI56vlHWggWqI0alLGoRBXk5adSeK0i39z/0y6LGkchDt+cumxUGlyYXjpr913WEknSg8WvoDuJEXtl04AtMkPqjoQBpP3QGvuiQRaGlxIJ0ovFv6Y4JlZW4asWWSsvhoc1+qLolXcGzGpCCMUE+Gt8OrLqnKjdFCS/+fuAeEfI2NEr4WElnYm+HeGZ9iIRRImkyHDmVGWpfk0uW9Mh234tGSpbwca51WjyXQ4wHhp0FBQ9fh1xhwn94PMJh21CUNUnUnCjWR+q1nS3/m253bJFYs+qemoW+xKN9rQfnb6q2WCOto5H6fLeO84pRFQaXJDftSLwWW8THf4SG93DKjT1uG/jXT0L9nGnqSiZ/GUFHiZ8ayBHq8IgC6zjwY2VDpQq0p/TMiTnkMBdi3zyuLosVZo7WjGj1qduKPCehx34Dfx8AGAF8AMN8xOtU6mbuumvtICjEQMd1mxVvKHn+gJ5GzbeF15PN6njNtioe8amzFdqXzAECwqzts1t2dtYn+pmoDmN5X9ZzznMEgWmG276kUD9nAXQDyDvEkK6ytCMKYhhf7/g8DZTsOqs/4wl2pRxj4VuUr6R2AfsRQFmiJtNz6qJLiLGye+bZe9hHemEy9waBfugYCCqYJYALe8RiaYkF1frq6IuFE+usgWu3x6s2BeSdAt6gi36IlUp8f7YrWcxUyDX0o8yEk1a50vGyRvtH8CRvCVZhGJP5E7ep9plaDTEM/BMDr+1W7tUT6D0eik5dBsfZHLhPEzbbAUVU0HiysGUlqZLgDARCwrwklDv+81AReBsU6oL8GZ2kt0X1aV+rmWowp6D4FwLvBI+FyrSvt9QlqyRjhWkjMQ5SNA2g78kTszpYqtGrzQ/bllotRynkGkCvKdYbLgQhYWukTiKap3g0MLwIE0JiD5evLd6VQ8vniRkvu1yjjhqAyPkkwCLizHkUo5VuuTXzpzRQB7gJ9xpc4FqtUS+4JA4IhVhWJ7oFXxtegyY3ROkKA3d/pBMhHdQS5Ps/IQKuFt0dUGWrFIysBjg8p400M9wdBAH/fcpCcHQQU9+YjgXUzo19bbmKhs5W7wpRow+mOltZqjMh0ROeCxfeKRC9pyb5dXp1ZAbRXe2ZaMnoIbV/qBQDPOwfIx9oOM9/uIZ4Ssu0dfmOVbLw5Lmx+FOALijR/HwDC81O/ADDY9uU1IvqaivyshgM9r/jRLRl9CABMI7oGcGVWbCN/SUPirVdLTWZAWHF9d4nehc/bilhRqh8gL5p6gZlt/BKBvwxgYpHSl9WGtFHoyopsPLJM2HQilEz/ulyzBcnYQADAHc0TLVvpHf4UAJj438JdfWU7vZuxloUg3uVRvQEA/WDsYMIvBehwntGoCJ5jM5YQsATulD0HYX/A2QtIUr+cqY0349EfgPnzjvEjKk9rrdSK1orpX2fCP9doiw3gZi2Rdn/iSFK3nFkHYuS9AtYZJo5XrI4IJdPf8srKquA0M18nnWf8ccaBwl2Hu+BVHUGVqyMIYDWR+msw/h3VxymP2XllYTjZ5z7LLKl7hq9EM3s9hf4sG4/Oq6SIAFtLpm8jwpUAV9jZpndA2EoCV2iJ9FUyqxq/DOsPxItbG6138z0oNL8cGsB3tWT61moUZ+PNccHiWgZFCZjB4GNE9Dps3hvCtMf9tviX1DeuBlNmXP8uGF90iH+vhqmF9qScNVeS8xzXZioPlMW46+Gz7OsTiJLzC5cDhbtSBwE85XGt3IOSuPAsLGT2Ks7D+7Px5riHXHIe4+lAGqY+DOAtp5zsAJtLSc4JPB2IkkmTQc5S4xeY6LmzYJNkHFGyzW/GmHmxgLKPQP8NwibZL1lSNUG1spNIJBKJRCKRSCQSiUQikUgkEolEIpFIJBKJRCKRSCQSiUQikUgkEolEIpFIJBKJRCKRSCQSiUQikUgkEolEIpFIJOOE/weptMzr5TiySAAAAABJRU5ErkJggg==);\\"></view>
</view>"
`;
